### PR TITLE
don't nag people in aftercore about a lack of batteries when the mall is a thing which exists.

### PR DESCRIPTION
**Pull request by Malibu-Stacey**
also stillsuit recommendation for 11 advs is also really only applicable in-run and I fixed the formatting & mixed tabs & spaces in both files.

### CODE REVIEW

File: Source/relay/TourGuide/Items of the Month/2021/Potted Power Plant.ash [hunk 1]
```diff
@@ -2,52 +2,51 @@
...
-	string batteriesToHarvest = (get_property("_pottedPowerPlant"));
-	// Entries
-	if (batteriesToHarvest != "0,0,0,0,0,0,0")
-	{
-		string main_title = "Power plant batteries";
-		string [int] harvest;
-		harvest.listAppend("Harvest your potted power plant batteries.");
-		resource_entries.listAppend(ChecklistEntryMake("__item potted power plant", "inv_use.php?pwd=" + my_hash() + "&whichitem=10738", ChecklistSubentryMake(main_title, "", harvest), 1));
-	}
+  if (!lookupItem("potted power plant").have())
```
Comment (docrostov) [inline Source/relay/TourGuide/Items of the Month/2021/Potted Power Plant.ash:5]: thank you for fixing these tabs. at some point we should set up a prettier linter for this repo.

File: Source/relay/TourGuide/Items of the Month/2022/Tiny Stillsuit.ash [hunk 2]
```diff
@@ -2,78 +2,89 @@
...
+    sweatCalcSweat = 43;
+    sweatCalcAdvs = 4;
+  }
+  else if (fam_sweat_o_meter >= 10)
+  {
+    sweatCalcSweat = 23;
+    sweatCalcAdvs = 3;
+  }
+  
+  if (fam_sweat_o_meter > 358 && __misc_state["in run"]) {
```
Comment (docrostov) [inline Source/relay/TourGuide/Items of the Month/2022/Tiny Stillsuit.ash:64]: the only thing i'd note here is that we don't have adv totals in the messy if/else above beyond 10 advs, so the final else here is going to mislead anyone using this tile in aftercore. might want to just remove the "you should probably guzzle your sweat now" line and replace it with something along the lines of "You will get > 10 adventures by guzzling sweat now." to handle the fact that tes could not figure out the root function stuff and there appear to be weird ash bugs with it.

**Review by docrostov — COMMENTED** [review]
not approving quite yet, would like some thoughts on the stillsuit thing. i think we need to slightly change up the way that limiter if-else works maybe.

File: Source/relay/TourGuide/Items of the Month/2022/Tiny Stillsuit.ash (hunk 2 above)
Comment (Malibu-Stacey) [reply Source/relay/TourGuide/Items of the Month/2022/Tiny Stillsuit.ash:16]: Fixed in latest. Thanks to @horrible-little-slime for his maths help as my brain is currently undergoing maintenance.

**Review by docrostov — APPROVED** [review]
looks good. thanks stacey!

**Pull request merged**